examples/wms.ipynb

```diff
@@ -142,10 +160,11 @@
     "ax = plt.axes(projection=ccrs.PlateCarree())\n",
     "\n",
     "ax.add_wms(wms,\n",
-    "           layers=[str(workflow)],\n",
+    "           layers=[str(workflow)],           \n",
     "           wms_kwargs={\n",
     "                'time': urllib.parse.quote(bbox.time_str),\n",
-    "                'crs': bbox.srs\n",
+    "                'crs': bbox.srs,\n",
+    "                'exceptions': 'application/json'\n",
```

Comment:
is this required or optional?

Reply:
owslib sets it explicitly to XML, so I override it here.

Reply:
i guess we should we set that on a global level for wms requests to the geo engine?

Reply:
I added support for XML in the backend and removed the format here c16e108fc21705b63fcdfad3645128e4b669a0f3